event_parsing.go

```diff
@@ -23,6 +23,7 @@ const (
 	EventTypeSystemHook    EventType = "System Hook"
 	EventTypeTagPush       EventType = "Tag Push Hook"
 	EventTypeWikiPage      EventType = "Wiki Page Hook"
+	EventTypeDeployment    EventType = "Deployment Hook"
```

Comment:
Please keep this list alphabetically ordered (ignoring the `EventConfidential*` entries)